src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+@import '../utils/variables';
+
+@keyframes hoverTransform {
+  from {
+    transform: translateY(0);
+  }
+
+  to {
+    transform: translateY(-20px);
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: (($day-size + $gap-columns) * $columns);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: '#{$number}';
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $bg-hover;
+      transform: translateY(-20px);
+      animation: hoverTransform 500ms ease-in-out;
```

Comment:
Remove the `transform` property from the `:hover` state, as it conflicts with the keyframes animation. If you decide to use `transition`, apply it to the `transform` property outside of the `:hover` state.